internal/scheduler/schedulerobjects/schedulerobjects.proto

```diff
@@ -31,6 +31,8 @@ message Node {
     string id = 1;
     // Kubernetes node name. Not guaranteed to be unique across clusters.
     string name = 12;
+    // Keys by which nodes are sorted in the nodeDb.
+    repeated bytes nodeDbKeys = 18;
```

Comment:
We now compute the index keys for the node once on upsert and store them on the node. Instead of re-computing these keys every time we iterate over the nodes, which caused performance issues.